MANIFEST.in

```diff
@@ -23,6 +23,7 @@ recursive-include helm *.helmignore
 recursive-include helm *.lock
 recursive-include benchmark *.py
 recursive-include scripts *.sh
+recursive-include demo *.yaml
```

Comment:
Hmm, to name the directory `demo` could mislead people to think that this will contain some largish demo material. I'd rather name it something like "configurations" or "kind", after the subject. However, I'm also not very fond of creating many new directories in the root of a Python package, such as `benchmark` and now `demo`... Shall we take this occasion to muse about a nicer place for these "etc" kind of files?

- e.g. locust stuff move to "tests" (and make sure is inactive)
- e.g. kind config move do "docs" (to reuse existing one?) and name it `kind-example.yaml` (since localhost is not actually needed?)
- if we shall have some more files for Amazon/Google clouds, perhaps name it "config"?  (reminds me of "helm/configurations" but obviously it would not be good to store these non-helm files there.
- ... or just name it more generally "etc"?

Just thinking out loud, haven't put too much thought to it yet...